### ISSUE TITLE
print to html，then print to pdf，code shows different in Chrome and Edge

### ISSUE THREAD

**Issue by Yeba** [neilsustc/vscode-markdown#248]
There is a paragraph of code like this：

<center>

# hello

</center>

``` matlab
% Acc,NMI,RI,Precision,Recall,Error_rate,specificity,F_value,Jaccard,E_value
function [Accuracy,NMI_value,A_correct,P_correct,R_correct,Error_rate,Spe_value,F_value,J_value,E_value,array_,correct,ncorrect]=self_Evaluation(Idx,label) %返回正确率以及正确个数

    [Label_H, Label_L]=size(label);
    if(Label_H==1)
        label=label';
        [Label_H,Label_L]=size(label);
    end

```

### print in Chrome

![pchrome](https://user-images.githubusercontent.com/18240596/43385601-babcfd80-9413-11e8-868e-de0425ebb695.jpg)
### print in Edge
![pedge](https://user-images.githubusercontent.com/18240596/43385610-c29f77f8-9413-11e8-9ab3-c8ad67cfd4c4.jpg)

### question
- why the \<pre class="hljs"\>  in  Chrome can not change line automatically？
- waht's more, the line below the \# maybe too black, I think grey is better.

### ps
Thank you for the support of math.  :) 👍

**Comment by neilsustc** [comment]
Edge also doesn't wrap line on my laptop

![image](https://user-images.githubusercontent.com/7588612/43392685-04664ad2-9427-11e8-9c0b-0004661a67fd.png)

You can add your own rule `white-space: pre-wrap` to change the default behavior

![image](https://user-images.githubusercontent.com/7588612/43393005-2498266c-9428-11e8-9e3c-4d7fdacf462f.png)

---

Again, you can apply `border-color: gray` to `h1` tags (overriding the vscode default styles)

---

In summary, you need a CSS file containing

```css
pre {
    white-space: pre-wrap;
}

h1 {
    border-color: gray;
}
```

and then apply it using vscode setting `markdown.styles`

**Comment by Yeba** [comment]
It works, thanks!
